Sources/Bot/Services/DispatchService.swift

```diff
@@ -85,8 +88,7 @@ public final class DispatchService {
             } else {
                 let newService = makeMergeService(
                     targetBranch: targetBranch,
-                    scheduler: self.scheduler,
-                    initialPullRequests: [event.pullRequestMetadata.reference]
+                    scheduler: self.scheduler
```

Comment:
ℹ️ Note: the event is now just forwarded (see `mergeService.pullRequestChangesObserver.send(…)` below) in *all* cases (existing OR newly-created service) and is processed by the `MergeService` in both cases, which means we don't need to pass it to `initialPullRequests` anymore and make a special case. Instead this is now consistent as we _always_ just forward the GH event (wether the MergeService already existed for that branch or we spawn a new one)

By contrast, in the previous implementation:
 * the event was ignored when the `MergeService` was in `.starting` state which is why we passed it in `initialPullRequests` in the case a MS didn't exist yet and we spawned a new one…
 * but that former logic was not consistent and made a special case out of this, leading to the inconsistencies in how PRs were treated by existing vs newly-spawed `MergeServices`…  which is what led to that IOSP-467 bug in the end